activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -1159,19 +1169,20 @@ def add_index_options(table_name, column_name, comment: nil, **options) # :nodoc
         end
 
         using = "USING #{options[:using]}" if options[:using].present?
+        if_not_exists = "IF NOT EXISTS " if options[:if_not_exists].present?
```

Comment:
what if `:if_not_exists` = `false` ?

Reply:
Ohhhhh. You're definitely right, let me add that case and a test for it.